### PR TITLE
[Fix][Docs] Update jdbc doc and kafka doc

### PR DESCRIPTION
**Pull request by chaorongzhi**
## Purpose of this pull request

The doc of next vsersion has some problems, this pull request fixing some issues with the jdbc and kafka doc.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/Jdbc.md [hunk 1]
```diff
@@ -201,12 +201,13 @@ CDC(Change data capture) event
...
-        url = "jdbc:mysql://localhost:3306/test"
+        url = "jdbc:mysql://localhost:3306"
         driver = "com.mysql.cj.jdbc.Driver"
         user = "root"
         password = "123456"
         
-        table = sink_table
+        database = "sink_database"
+        table = "sink_table"
         primary_keys = ["key1", "key2", ...]
```
Comment (liugddx) [inline docs/en/connector-v2/sink/Jdbc.md:211]: What's the advantage of this change?

Comment (chaorongzhi) [reply docs/en/connector-v2/sink/Jdbc.md:211]: The parameters of the dev branch connector have been changed, but not in the doc examples.These two changes were discovered when I was using the dev branch of SeaTunnel.

Comment (chaorongzhi) [reply docs/en/connector-v2/sink/Jdbc.md:211]: > What's the advantage of this change?

The parameters of the dev branch connector have been changed, but not in the doc examples.These two changes were discovered when I was using the dev branch of SeaTunnel.

**Review by liugddx — APPROVED** [review]
LGTM, thansk for your contribution.

**Comment by chaorongzhi** [comment]
> LGTM, thansk for your contribution.

It's my pleasure.

**Review by lightzhao — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Pull request merged**